certificate/views.py

```diff
@@ -2544,7 +2542,7 @@ def fossee_internship16_cerificate_download(request):
         file_name = file_name.replace('.', '')
         try:
             old_user = Certificate.objects.get(email=email, serial_no=serial_no)
-            qrcode = 'Verify at: http://fossee.in/certificates/verify/{0} '.format(old_user.short_key)
+            qrcode = 'http://fossee.in/certificates/verify/{0} '.format(old_user.short_key)
```

Comment:
Should this be done for all the certificates?
Or may be it's fine for this one only.

Reply:
there are currently 2 active methods for the certificate. I will remove that text('Verify at') from both and push